### ISSUE TITLE
Bug: Random Manual Order Window failure

### ISSUE THREAD

**Issue by nancikennedy** [BloodDragon2580/Portable#10]
Occasionally, the window that pops up when clicking  "Manually Change the Spell Order for xxx" fails to properly populate. Rows show up in the window, but no contents. Reloading a few times usually fixed this.

**Comment by BloodDragon2580** [comment]
Check this: https://github.com/BloodDragon2580/Portable/commit/a488121d6837823e1d21dad36da3105c19605955

**Comment by nancikennedy** [comment]
I've got this downloaded and running, but no idea if/when it might bug out randomly if this doesn't fix it...but I will be vigilant about checking it periodically

**Comment by nancikennedy** [comment]
This is still happening. I was able to replicate it this time. Reloading does "fix" it until a full exit of the game.
Steps:
Launched client
Logged into char
Open Portable and click on the Manual Reorder button
Get this:


![Portable Manual Ordering WIndow](https://user-images.githubusercontent.com/8207666/199387980-dcb5c962-0ad1-4d02-a9b0-13b397c90f6f.JPG)

Reloaded UI once, and now the window is functioning properly

![Portable Manual Updated](https://user-images.githubusercontent.com/8207666/199388364-951c5631-2812-4e6b-b185-7c727ea3bf33.JPG)

Completely exit the client, and see that all the above steps are repeated.

- Is this, therefore, maybe something to do with retrieving the relevant information from the server the first time?

**Comment by BloodDragon2580** [comment]
What addons you use at this time? On my test Account there was no Problem with the order screen.

**Comment by nancikennedy** [comment]
Sorry I didn't confirm that I tested this with all my addons enabled and also with only Portable enabled. The results were almost exactly the same.

It was with only Portable enabled that the Valdrakken icon is covered over by the icon for the spell that it is stacking with. The text stacking stays that same with the Valdrakken text in the background. The screen caps are from the tests with all my addons enabled, as the Portable solo tests had such subtle changes that I felt it wasn't important to include them.

**Comment by BloodDragon2580** [comment]
ok thx

**Comment by BloodDragon2580** [comment]
This addon fucked up my brain^^

**Comment by nancikennedy** [comment]
Yeah, it's a bit of a puzzle. I am now double checking my tests, just in case.

**Comment by nancikennedy** [comment]
I tracked this down to a conflict with another addon. It was a private one from a former guildmate and there were a few orphan files in it that were out of date but not prompting any actual errors. I must not have made sure this was turned off when I tested with everything but Portable loaded. It's not an addon that's available anymore, as they have quit WoW entirely, so I would close this bug report as a non-issue. The other bug with icon layouts was NOT similarly fixed by disabling/removing this addon.

This also explains why this was sporadic, as I normally have that old addon turned off.
